sources/PTYSession.m

```diff
@@ -3070,30 +3073,15 @@ - (void)setBlend:(float)blendVal {
     [_textview setBlend:blendVal];
 }
 
-- (BOOL)antiIdle
-{
-    return _antiIdleTimer ? YES : NO;
-}
-
-- (void)setAntiIdle:(BOOL)set
-{
-    if (set == [self antiIdle]) {
```

Comment:
I want to keep the checkbox that enables/disables the feature.
 * Using 0 to disable it is more unintuitive than having a checkbox, and requires extra wording on the preference panel.
 * It really is a dangerous feature and has been the cause of many bug reports from confused users, so the warning message is important to keep.